specs/loginServices.spec.ts

```diff
@@ -0,0 +1,18 @@
+
+import { mockPrismaClient } from './mocks/functionsMocks';
+import {checkUser} from '../src/services/loginServices';
+
+describe('Login Services', () => {
+  describe('When checkUser', () => {
+    it('should return an user if find one', async () => {
+      const user = await checkUser('user@example.com', mockPrismaClient);
+      expect(user?.name).toBe('User');
+      expect(user?.email).toBe('user@example.com');
+    });
+
+    it('should return null if not find one', async () => {
```

Comment:
Let's change the wording of the descriptions. 

> returns null if the user is not found

using "should" make it sound like we are not sure